code/game/atoms.dm

```diff
@@ -2140,14 +2143,32 @@
 						// Ctrl-LMB, Alt-LMB on one line...
 						if (lmb_rmb_line != "")
 							lmb_rmb_line += "<br>"
+							extra_lines++
 						if (SCREENTIP_CONTEXT_CTRL_LMB in context)
 							ctrl_lmb_alt_lmb_line += "[SCREENTIP_CONTEXT_CTRL_LMB]: [context[SCREENTIP_CONTEXT_CTRL_LMB]]"
 						if (SCREENTIP_CONTEXT_ALT_LMB in context)
 							if (ctrl_lmb_alt_lmb_line != "")
 								ctrl_lmb_alt_lmb_line += " | "
 							ctrl_lmb_alt_lmb_line += "[SCREENTIP_CONTEXT_ALT_LMB]: [context[SCREENTIP_CONTEXT_ALT_LMB]]"
 
-						extra_context = "<br><span style='font-size: 7px'>[lmb_rmb_line][ctrl_lmb_alt_lmb_line]</span>"
+						// Shift-LMB, Ctrl-Shift-LMB on one line...
+						if (ctrl_lmb_alt_lmb_line != "")
+							ctrl_lmb_alt_lmb_line += "<br>"
+							extra_lines++
+						if (SCREENTIP_CONTEXT_SHIFT_LMB in context)
+							shift_lmb_ctrl_shift_lmb_line += "[SCREENTIP_CONTEXT_SHIFT_LMB]: [context[SCREENTIP_CONTEXT_SHIFT_LMB]]"
+						if (SCREENTIP_CONTEXT_CTRL_SHIFT_LMB in context)
+							if (shift_lmb_ctrl_shift_lmb_line != "")
+								shift_lmb_ctrl_shift_lmb_line += " | "
+							shift_lmb_ctrl_shift_lmb_line += "[SCREENTIP_CONTEXT_CTRL_SHIFT_LMB]: [context[SCREENTIP_CONTEXT_CTRL_SHIFT_LMB]]"
+
+						if (shift_lmb_ctrl_shift_lmb_line != "")
+							extra_lines++
+
+						if(extra_lines)
+							extra_context = "<br><span style='font-size: 7px'>[lmb_rmb_line][ctrl_lmb_alt_lmb_line][shift_lmb_ctrl_shift_lmb_line]</span>"
+							active_hud.screentip_text.maptext_y = -10 + (extra_lines - 1) * -9	//first extra line pushes atom name line up 10px, subsequent lines push it up 9px
+																								//this offsets that and keeps the first line in the same place
```

Comment:
Don't do this, we don't align code, including templates.

Just put both these comments before the line itself.